### PR TITLE
Removes RCDs from autolathes (Alternative to PR #9089 and #9087)

### PR DESCRIPTION
**Pull request by Tupinambis**
## About The Pull Request

Does what it says on the tin. Removes RCDs as printable items in the autolathe.

UPDATE: NO LONGER REMOVES RPDs 

## Why It's Good For The Game

RCDs are very powerful pieces of equipment that should certainly be powerful, and (I presume) they were originally intended as a quicker and more versatile tool to make engineer's lives easier and more efficient, AND give them a leg up on the common masses. Unfortunately, with the addition of them to the autolathe, their power is free to anyone with a few mats. It was inevitable that this power would be noticed by players, and result in Pull Requests to nerf RCDs as a whole. This just hurts engineers, and does nothing to consider the facts that RCDs should have been a tool to make playing engineer just that little bit more special. No non engineering role should have ever had easy access to a tool that effectively nullifies the need to carry a variety of mats and tools to deal with obstacles. Printable RCDs also make airlock electronics useless, emags less useful, armblades less useful, carrying a variety of mats pointless, and that's not ideal.

Simply put, RCDs are fine where they are, but they shouldn't be available to everyone (easily) and should be a limited resource. Any direct nerf to RCDs is just a nerf to engineering, and of all the roles on the station, engineering is probably one of the least deserving of nerfs.

## Changelog
:cl:
balance: removes RCD from the autolathe
/:cl:

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
This makes drone life hard
This nerfs lava land roles
This is a IC propblem
This is a bad idea
No reason to removal it other then `Easy to make, it shouldnt be easy` but no other way for said problems to be fixed

**Comment by Tupinambis** [comment]
> This makes drone life hard
> This nerfs lava land roles
> This is a IC propblem
> This is a bad idea
> No reason to removal it other then `Easy to make, it shouldnt be easy` but no other way for said problems to be fixed

The bottom three I disagree with, but for the first two, those can be addressed in a second PR, by giving droned RCD by default and by mapping RCDs onto the relevant ghost roles.

**Comment by KrabSpider** [comment]
>Printable RCDs also make airlock electronics useless

No one is going to use up to 20-25 metal and glass for to print a device that requires metal and glass — just to make a few doors. This is especially so if some dork is just decconing airlocks and replacing them afterwards, wasting a *large* amount of ammo in the process. Deconstruction mode uses the most ammo.

>emags less useful

You have more variety in what you can use an emag for, and they get results a lot faster than simply deconstructing the airlock via RCD. 

>carrying a variety of mats pointless

...What do you mean by this, OP?

**Comment by silicons** [comment]
no thank you sorry!
